lib/chartmogul/summary_all.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module ChartMogul
+  class SummaryAll < ChartMogul::Object
```

Comment:
Why are we calling it SummaryAll - Isn't the end point MetricsAll?

Reply:
Endpoint - yes. But it is only represents Summary object for MetricsAll endpoint, similar to Summary for other endpoints: https://github.com/chartmogul/chartmogul-ruby/blob/3291c3a21b7c28a9d27a93711728822b3b90108a/lib/chartmogul/summary.rb